app.js

```diff
@@ -25,10 +28,19 @@ app.get('/impersonations/current', async function (req, res, next) {
   const data = {
     type: 'impersonations',
     id: sessionId,
+    attributes: {
+      'original-session-roles': originalSessionRoles,
+    },
     relationships: {
       impersonates: {
         links: `/resources/${impersonatedResourceId}`,
         data: { type: 'resources', id: impersonatedResourceId },
+      },
+      'original-resource': {
```

Comment:
Same here, we could name this `original-account`.

Reply:
maybe to be in consistent with the fact that we are tailoring with acm/abb setup.